src/sentry/incidents/models.py

```diff
@@ -164,12 +164,10 @@ class Incident(Model):
     )
     type = models.PositiveSmallIntegerField()
     title = models.TextField()
+    sub_title = models.TextField()
     # When we suspect the incident actually started
     date_started = models.DateTimeField(default=timezone.now)
-    # When we actually detected the incident
-    date_detected = models.DateTimeField(default=timezone.now)
```

Comment:
we should make this column nullable in a separate migration that comes first

Reply:
Yeah, I agree with Steve here. As it is, we'll be removing the field from the DB during deploy while the web app is trying to write to it and we'll see errors.